cf_xarray/accessor.py

```diff
@@ -1167,10 +1164,7 @@ def coordinates(self) -> Dict[str, List[str]]:
         Dictionary of valid Coordinate names that can be used with ``__getitem__`` or ``.cf[key]``.
         Will be ("longitude", "latitude", "vertical", "time") or a subset thereof.
         """
-        vardict = {
-            key: apply_mapper(_get_axis_coord, self._obj, key, error=False)
-            for key in _COORD_NAMES
-        }
+        vardict = {key: _get_coords(self._obj, key) for key in _COORD_NAMES}
```

Comment:
We should update the docstring to say that it only searches in `.coords` similarly for .axes etc.

Reply:
What's the downside of treating coordinates as measures, so `_get_all` rather than `_get_coords`?

Reply:
Is there some weird thing where there's a "data variable" with "latitude" values but also a different latitude coordinate? Like a trajectory thing with "true value longitude" and "nominal longitude"? Should we distinguish between the two?

Reply:
Something like this?
 ```python
 _get_coords(obj, key) if _get_coords(obj, key) else _get_all(obj, key)
```

Reply:
mmm, actually the way it is right now is probably the best

Reply:
The first comment in #107 brings up this point.

The fundamental issue is whether `.axes` et al should returns results present only in coordinates or in all variables. With the methods, we can control this by using `_get_dims` and friends. With the properties we don't really have a good way of adding any user control. That  said, asking people to use `set(obj.cf.axes) & set(obj.cf.dims)` is really not too bad.

Maybe all properties should return names present in all variables (not just data variables), and we can add an FAQ showing people how to use set operations to pull out what's needed?